project2-beat-that/script.js

```diff
@@ -1,4 +1,272 @@
+// Generate 2 dice rolls for player 1
+// > return user the 2 dice rolls
+// > Allow player 1 to choose order of 2 dice rolls
+// > return the player's choice
+
+// 2nd version
+
+var comparingCombineDiceRolls = function () {
+  var max = Math.max.apply(null, playerScores);
+  var winningPlayer = playerScores.indexOf(max);
+  return winningPlayer;
+};
```

Comment:
`Math.max(...playerScores)` is the more modern way of obtaining the max value from an array of numbers

you can refer to these 2 links for more info:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max#examples
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

Reply:
also what about tie scenarios?

lets say my scores are [100, 50, 100]

Math.max will return 100 and indexOf will return 0, and your subsequent logic below will state that player 1 won